projects/angular/src/data/datagrid/providers/items.ts

```diff
@@ -24,8 +26,15 @@ export class Items<T = any> {
 
   /**
    * Tracking function to identify objects. Default is reference equality.
+   *
+   * @deprecated in v15 and scheduled for removal in v17
+   */
+  iteratorTrackBy: TrackByFunction<T> = (_index, item) => item;
```

Comment:
Please file a ticket to remove this function in v17 and add a reference (e.g. `TODO(CDE-[n])` in this code block.

Reply:
Good idea.

Reply:
Done.